### PR TITLE
Adds main files and emperor submodule

### PR DESCRIPTION
**Pull request by ElDeveloper**
I'm adding the files that we created using `npm run newapp`, note that I'm adding the emperor repo as a submodule, and finally this project is licensed under the BSD-2 license.

Most of the code in emperor.js was auto-generated, though I have now cleaned it up a bit and changed it to adhere to our usual coding guidelines.

---

Credit for this source goes to:

@squirrelo, @mortonjt, John Graham and Luc Renambot.

### CODE REVIEW

File: emperor [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit e9696d7310094227f9ed387259e85dd02785450b
```
Comment (antgonza) [inline emperor:1]: Is this needed?

**Comment by antgonza** [comment]
Should the emperor.png be the emperor logo?

File: emperor (hunk 1 above)
Comment (ElDeveloper) [inline emperor:1]: Yes, this is set up by git when you create the submodule.

**Comment by ElDeveloper** [comment]
Yes, I just changed that, looks much nicer!

**Pull request merged**